.github/workflows/parachain.yml

```diff
@@ -77,11 +77,17 @@ jobs:
       - name: cargo test
         uses: actions-rs/cargo@v1
         with:
-          command: tarpaulin
-          args: >-
-            --manifest-path parachain/Cargo.toml
-            --verbose --workspace
-            --exclude artemis,artemis-runtime
-            --exclude-files 'src/command.rs,*/mock.rs,*/test.rs'
-            --coveralls ${{ secrets.COVERALLS_REPO_TOKEN }}
-          toolchain: nightly-2021-01-10
+          command: test
+          toolchain: stable
+          args: --manifest-path parachain/Cargo.toml --workspace --exclude artemis
+      # - name: cargo test
```

Comment:
Tarpaulin is producing a strange error during `cargo test`, it looks like a bug. I disabled it temporarily because it seems obscure, but I can try and dig deeper into this.